xml/admin_caasp_object_storage.xml

```diff
@@ -1092,7 +1092,7 @@ kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml \
   </sect2>
  </sect1>
  <sect1 xml:id="ceph-object-store-user-crd">
-  <title>Ceph Object Store User CRD</title>
+  <title>&ceph; &objstore; User CRD</title>
```

Comment:
CRD appears throughout this diff, i just want to make sure the user knows what the acronym means, because i don't (but i havn't read the full version). I just hope it doesn't stand for 'credentials' or something.

Reply:
I believe we explain CRD at the beginning of the document, just these files are out of sync. I'll double check though